docs/pages/Configuration.md

```diff
@@ -465,3 +465,11 @@ Version 1.0 projects are supported by [UI5 CLI](https://github.com/SAP/ui5-cli)
 Initial version.
 
 Version 0.1 projects are compatible with [UI5 CLI](https://github.com/SAP/ui5-cli) v0.0.1 and above.
+
+## Validation / IDE support
+
+Starting with specification version 2.0 the configuration is validated according to a JSON schema.  
```

Comment:
This confuses me - are you perhaps talking about UI5 Tooling version 2.0 ? Then we should say so.

Reply:
No. The specification version is something different than the UI5 Tooling version. One is the version of a projects configuration, the other is the version of the tool you are using.

Coincidentally the latest versions of both are equal (2.0). But they can be different. Also see https://sap.github.io/ui5-tooling/pages/Configuration/#specification-versions

We only validate projects with a configuration defining "specVersion: 2.0"

Reply:
OK, thanks. Yes, the equality of version numbers confused me here. Do you think it would make sense to include the link?

Reply:
Yes, since we move it to a different chapter of the document a link makes sense.

Reply:
I've added a link to specVersion 2.0